### PR TITLE
#989 fix

### PR DESCRIPTION
**Pull request by jafdy**
I haven't had a chance to test it and its not very good at the moment but it will make "what happened to Rand" mission (FW Pug 1) make more sense with a jump drive.
I will extend the effect to the rest of the reconciliation story line at some point.
Edit: tested it and it works.

### CODE REVIEW

File: data/free worlds reconciliation.txt [hunk 1]
```diff
@@ -924,6 +924,77 @@ mission "FW Pug 1 Arabia"
...
+		
+			label cut2
+			`	"I'd think that would be obvious. With the hyperspace links cut, there's no way for anyone from outside to assist us. And they struck at a part of human space without much in the way of militias or bounty hunters or Navy protection. And the few half-decent warships that happened to be in system when they attacked have already been destroyed trying to fight them."`
+				goto end
+		
+			label end
+			`	"Where did they come from originally?" you ask.`
+			`	"From a star system that was always a neighbor of ours, but wasn't linked to us or to any other part of human space. We always just assumed it was an uninhabited system, but it turns out they were there all along, biding their time, waiting for a good opportunity to attack us."`
+			`	"Do you know how the Pug cut you off?`
+			`	"Look I'm no scientist, I have no idea how Hyperspace links work. Maybe someone on the other end can help you.`
```
Comment (toilethinges) [inline data/free worlds reconciliation.txt]: Missing comma, and an extra 'and' could be inserted to make it look a bit better. "Look, I'm no scientist, and I have no idea how hyperspace links work." Also, missing end quotation.

Comment (jafdy) [inline data/free worlds reconciliation.txt]: My grammar is not a strong point of mine.
Thanks.

File: data/free worlds reconciliation.txt [hunk 2]
```diff
@@ -979,6 +1053,18 @@ mission "FW Pug 2"
 			
 			`	"We don't," says JJ, "but this all certainly plays out very well for them; their section of the galaxy has suddenly become a very defensible enclave. And the timing of that fleet leaving, and then the link closing - it's too much to be a coincidence. But we can't learn anything for sure by staying here talking; you must go to Earth."`
 				accept
+
+			label "know about pug"
+			`By the time you arrive, Alondo and Freya have joined JJ and Katya on New Tibet; the entire Council is present. "Here's the story," says JJ. "A few weeks ago, hyperspace links on the edge of Syndicate space began collapsing. We weren't sure why."`
+			`	"I know why." you interrupt. The silence this statment leaves behind is deafning.`
```
Comment (toilethinges) [inline data/free worlds reconciliation.txt]: Deafening.*

File: data/free worlds reconciliation.txt [hunk 3]
```diff
@@ -979,6 +1053,18 @@ mission "FW Pug 2"
...
+			`	Evetuly Katya asks "<first>, is it the Syndicate?"`
+			`	"No, it is an alien race called the pug. Thay say thay are there to promote peace but I don't know their true intentions."`
```
Comment (toilethinges) [inline data/free worlds reconciliation.txt]: are there to promote peace, but I don't know

**Comment by jafdy** [comment]
Most of what is still needed is a copy and paste job. I'll try to get it done tomorrow.

**Comment by jafdy** [comment]
It's done now. I'll say when I've tested it. @endless-sky Thoughts?

**Comment by jafdy** [comment]
Tested and ready to merge.

File: data/free worlds reconciliation.txt [hunk 4]
```diff
@@ -1520,6 +1621,368 @@ mission "FW Pug 3C"
...
+	to offer
+		has "FW Pug 2: done"
+		has "fw know about pug"
+	on fail
+		dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+
+	on offer
+		conversation
+			`A palpable sense of fear and near-panic hangs over the spaceport on Earth. At any minute, this system that serves as the very hub of human government could disappear into the same unknown abyss that has swallowed up its neighbors.`
+			`	Parliament is eager now to meet with you and to receive whatever assistance you may be able to offer. As Alondo stands in front of the chamber telling the members of Parliament what you already know and asking questions about the disappearance of the Syndicate fleet, they answer him with respect and deference, far different from how you have been treated here in the past. You learn that although one link remains open to Syndicate space, no one has dared to travel through it. Given the pace at which the other links were cut, they had feared that the one remaining link will disappear at any time.`
```
Comment (Gkramme) [inline data/free worlds reconciliation.txt]: The last Jump Drive is not capitalised.

Comment (jafdy) [inline data/free worlds reconciliation.txt]: It's a bit odd. The original missions use no caps for jump drives but they are names so should be with caps (Jump Drives). If I want to get consistency then ether my pull should have them all lower case or I will have to go through the FWR and FWC files and move everything to upper case.

**Comment by Isaacssv552** [comment]
The new missions don't give an option to say that I looted my jump drive off the Quarg. It seems like a fairly natural rejoinder to the statement that "only the Quarg use jump drives".

**Comment by jafdy** [comment]
First question. No it doesn't. I haven't been able to think of a good way to split the difference between someone who has got a Quarg Drive and someone who got a Korath Drive so I assumed that you probably got your first one (and the one on your flagship) from the Korath. Also no one would believe that you disabled a Quarg ship anyway.

Second question. No. The story changes if you visit Rand during "What happened to Rand?" Without ripping the player's Jump Drive out and sticking it back in there is no way to set mission requirements to having outfits and even if there where the Drive on your flagship is the only one that matters not ones parked on other planets or in your cargo.

**Comment by Isaacssv552** [comment]
I figured that was probably the case. Thanks for adding the Quarg option; it needs some editing but resolves the issue.

**Comment by jafdy** [comment]
This story writing lark is harder than it looks. I'm glad you like it

File: data/free worlds reconciliation.txt [hunk 5]
```diff
@@ -1554,6 +1560,643 @@ mission "FW Pug 3C"
...
+			`	One of the Midgard representatives stands. "The Navy has sensors that could confirm that Captain <last>'s ship arrived using a Jump Drive." The Speaker asks you to leave while Parliament hears what the Navy knows and decides what should be done.`
+			`	Outside Alondo looks stressed, "Even when they have all the facts they still can't decide things quickly. Well I suppose our Senate is not much better."`
+			choice
+				`	"Politicians always decide things slowly, and these ones are running scared."`
+					goto scared
+				`	"It can be tiring sometimes but we can't run everything."`
+					goto run
+
+			label scared
+			`	At that moment a Navy admiral hurries past looking stressed. "Your right there," says Alondo, "must be all that public exposure they get making them far to careful."`
```
Comment (toilethinges) [inline data/free worlds reconciliation.txt]: This line: Your - You're*
1791: Karath - Korath*, planet - system(?)*
1783: North - East/North-East(?)*

File: data/free worlds reconciliation.txt (hunk 4 above)
Comment (jafdy) [inline data/free worlds reconciliation.txt]: @Gkramme 
Ok. Every mention of Jump Drive is now in caps.
Could you get rid of the line comment.

**Comment by jafdy** [comment]
@endless-sky do you know what the conflict(s) is/are?

**Comment by Gkramme** [comment]
You have to merge the branch to update the commits?

**Comment by jafdy** [comment]
Well now it's broken and I have no idea how to fix this.

**Comment by jafdy** [comment]
Well now it's broken and I have no idea how to fix this.

**Pull request closed without merging**